library/psa_crypto_se.c

```diff
@@ -32,13 +32,53 @@
 
 #include "psa_crypto_se.h"
 
-typedef struct
+/****************************************************************/
+/* Driver lookup */
+/****************************************************************/
+
+typedef struct psa_se_drv_table_entry_s
 {
     psa_key_lifetime_t lifetime;
     const psa_drv_se_t *methods;
-} method_table_entry_t;
+} psa_se_drv_table_entry_t;
+
+static psa_se_drv_table_entry_t driver_table[PSA_MAX_SE_DRIVERS];
+
+const psa_se_drv_table_entry_t *psa_get_se_driver_entry(
+    psa_key_lifetime_t lifetime )
+{
+    size_t i;
+    if( lifetime == 0 )
```

Comment:
I have the same remark as in a different PR - I would suggest swapping zero with a defined constant to signal what does this value mean.

Reply:
Here 0 means no entry. I'll add a comment to explain.